pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -137,6 +197,7 @@ in stdenv.mkDerivation {
 
   preConfigure = ''
     mesonFlagsArray+=(-Dntp-servers="0.nixos.pool.ntp.org 1.nixos.pool.ntp.org 2.nixos.pool.ntp.org 3.nixos.pool.ntp.org")
+    export AR="gcc-ar"  # For LTO
```

Comment:
This might need a prefix for cross compiling:

```
export AR="${stdenv.hostPlatform.config}-gcc-ar"
```

Reply:
Oh, yes, of course! Thanks.